tedana/workflows/tedana.py

```diff
@@ -337,11 +327,10 @@ def tedana_workflow(data, tes, mask=None, mixm=None, ctab=None, manacc=None,
                                                 t2s, t2sG, stabilize, ref_img,
                                                 tes=tes, kdaw=kdaw, rdaw=rdaw,
                                                 ste=ste, wvpca=wvpca)
-        # Perform ICA on dimensionally reduced data (*without* thermal noise)
-        mmix_orig, fixed_seed = decomposition.tedica(n_components, dd, conv, fixed_seed,
-                                                     cost=initcost, final_cost=finalcost,
-                                                     verbose=debug)
+        mmix_orig = decomposition.tedica(n_components, dd, conv, fixed_seed,
```

Comment:
The updated version of tedica is still returning `fixed_seed`, so you'll need to grab that here as well.

```suggestion
        mmix_orig, fixed_seed = decomposition.tedica(n_components, dd, conv, fixed_seed,
```

Reply:
That was the whole merge error !! Thanks for catching it :tada: